src/components/Image/Image.tsx

```diff
@@ -24,11 +30,13 @@ export interface ImageProps extends TPAComponentProps {
   onError?: React.EventHandler<React.SyntheticEvent>;
   /** Specifies how the image is resized to fit its container */
   resize?: 'contain' | 'cover';
+  // Specifies the proportional relationship between width and height
+  aspectRatio?: 'square' | 'cinema' | 'landscape' | number;
   /** An experience to set while the image is fetched and loaded  */
   loadingBehavior?: 'none' | 'blur';
 }
 
-type DefaultProps = Pick<ImageProps, 'resize'>;
+type DefaultProps = Pick<ImageProps, 'resize' | 'aspectRatio'>;
```

Comment:
I meant `Pick`, which is util of TypeScript to pick only specific fields from an interface (in this case - ImageProps)  :)

Otherwise I would have to duplicate the code and declared them again (not a big deal BTW, but the current one is just cleaner)

Reply:
NIce, good to know:)